src/client/use-sse.ts

```diff
@@ -178,6 +178,10 @@ export function useSSE<T = any>({
       return destructor;
     };
 
+    if (url === '') {
+      return;
+    }
```

Comment:
Thank you for contributing! I understand the issue, however, there is still a small error in the proposed solution.

The problem is that the URL could change while data is still being received, and a destructor was not called within the handleError function because a reconnect process was initiated. In this scenario, a connection would remain in the `open` or `connecting` state. Therefore, the connection must also be explicitly closed.

```suggestion
    if (url === '') {
      if (connectionState !== 'closed') {
        close();
      }

      return;
    }
```